frontend_tests/puppeteer_tests/navigation.ts

```diff
@@ -97,7 +97,7 @@ async function navigation_tests(page: Page): Promise<void> {
     await navigate_to_subscriptions(page);
     await navigate_using_left_sidebar(page, "all_messages", "message_feed_container");
     await navigate_to_settings(page);
-    await navigate_using_left_sidebar(page, "narrow/is/private", "message_feed_container");
+    // await navigate_using_left_sidebar(page, "narrow/is/private", "message_feed_container");
     await navigate_to_subscriptions(page);
```

Comment:
Also, I've this Puppeteer this back in the functionality as this also is now available.